src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -336,7 +339,7 @@ export class InstructorCourseEditPageComponent implements OnInit {
    */
   cancelEditingInstructor(index: number): void {
     const panelDetail: InstructorEditPanelDetail = this.instructorDetailPanels[index];
-    panelDetail.editPanel = this.getInstructorEditPanelModel(panelDetail.originalInstructor);
+    panelDetail.editPanel = panelDetail.originalPanel;
```

Comment:
I think this is shadow copy, which means that the second time you click cancel and come again will reproduce the issue you mentioned again?